components/pages/explorer/MobileWarning.tsx

```diff
@@ -0,0 +1,99 @@
+/*
+ *
+ * Copyright (c) 2023 The Ontario Institute for Cancer Research. All rights reserved
+ *
+ *  This program and the accompanying materials are made available under the terms of
+ *  the GNU Affero General Public License v3.0. You should have received a copy of the
+ *  GNU Affero General Public License along with this program.
+ *   If not, see <http://www.gnu.org/licenses/>.
+ *
+ *  THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS "AS IS" AND ANY
+ *  EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT LIMITED TO, THE IMPLIED WARRANTIES
+ *  OF MERCHANTABILITY AND FITNESS FOR A PARTICULAR PURPOSE ARE DISCLAIMED. IN NO EVENT
+ *  SHALL THE COPYRIGHT HOLDER OR CONTRIBUTORS BE LIABLE FOR ANY DIRECT, INDIRECT,
+ *  INCIDENTAL, SPECIAL, EXEMPLARY, OR CONSEQUENTIAL DAMAGES (INCLUDING, BUT NOT LIMITED
+ *  TO, PROCUREMENT OF SUBSTITUTE GOODS OR SERVICES; LOSS OF USE, DATA, OR PROFITS;
+ *  OR BUSINESS INTERRUPTION) HOWEVER CAUSED AND ON ANY THEORY OF LIABILITY, WHETHER
+ *  IN CONTRACT, STRICT LIABILITY, OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN
+ *  ANY WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+ *
+ */
+
+import IconButton from '@/components/IconButton';
+import { css, useTheme } from '@emotion/react';
+import { useState } from 'react';
+import { getConfig } from '../../../global/config';
+import { Dismiss as DismissIcon, Error as ErrorIcon } from '../../theme/icons';
+
+const MobileWarning = () => {
+	const theme = useTheme();
+	const [showMobileWarning, setShowMobileWarning] = useState<boolean>(true);
+	const hideMobileWarning = () => setShowMobileWarning(false);
+	const { NEXT_PUBLIC_SHOW_MOBILE_WARNING } = getConfig();
+
+	return showMobileWarning && NEXT_PUBLIC_SHOW_MOBILE_WARNING ? (
```

Comment:
`NEXT_PUBLIC_SHOW_MOBILE_WARNING` is already checked in PageContent
```suggestion

	return showMobileWarning ? (
```